app/Http/Controllers/RESTController.php

```diff
@@ -227,31 +226,28 @@ abstract protected function getResource();
      */
     protected function getRouteParams(Request $request)
     {
-        return $request->route()[2];
```

Comment:
@webong @Mh-Asmi funny little change. What do you think happened here? Why did we have a hardcoded array index of [2]? 🤔 how is the change equivalent?

Reply:
ahh interesting :) Thank you 🙏